ktlint-ruleset-standard/src/main/kotlin/com/pinterest/ktlint/ruleset/standard/SpacingAroundCurlyRule.kt

```diff
@@ -67,6 +70,18 @@ class SpacingAroundCurlyRule : Rule("curly-spacing") {
                 ) {
                     emit(node.startOffset, "Unexpected newline before \"${node.text}\"", true)
                     if (autoCorrect) {
+                        val eolCommentExists = prevLeaf.prevLeaf()?.let {
+                            it is PsiComment && it.elementType == EOL_COMMENT
```

Comment:
Now when I'm thinking more about it - would `/* a comment */` also have `EOL_COMMENT` type actually? I think this should also be allowed.. sorry for not spotting it earlier

Reply:
I'm not sure that end-of-line and block comments should be handled the same here. The original bug, #728, addressed the fact that an end-of-line comment caused the output you get from running ktlint to no longer be compilable Kotlin code which I think most people would find undesirable regardless of how the code is styled. In the case of a block comment, the code generated from the current ktlint will still be compilable. If we are going to handle block comments, do we also need to handle various styles of multi-line comments that might be found? If you think we should handle block comments can you provide examples on the desired before/after code for each case? I'd be happy to make the changes but I'm not 100% clear on what the goal is.

Reply:
True, I think you're right, handling different styles of comments is probably an overkill here.. great, then it looks good 👍